### ISSUE TITLE
Any plans on further development

### ISSUE THREAD

**Issue by ludmiloff** [tweedjs/tweed#2]
First of - thank you for your library, it's design is really interesting and the library ecosystem is well documented.
Now I'd like to ask do you have any plans for further development - snabbdom update, Webpack  update and adding more features.

Thank you.

**Comment by emilniklas** [comment]
Hello @ludmiloff 

Thanks for reaching out. I've not been using Tweed myself for the past months, because I haven't had any side projects fit for it, and we're doing React at work. Thus, I'm unsure where to take Tweed next. I don't know what the biggest pain points are going to be, so I'm hesitant to introduce more features "just because". I like that Tweed is really lightweight and streamlined.

I assume at least part of the reason for your question is to probe whether or not there's any commitment to the project.

There is.

I'd very much like to keep exploring this space. However, I feel like the JS packaging story is somewhat close to going through some changes. In particular, see this post: https://babeljs.io/blog/2018/06/26/on-consuming-and-publishing-es2015+-packages

I think there's a lot of improvement to be done in packaging Tweed. Right now, `babel-runtime` is included as a dependency of version `*`, to be compatible to whatever version the consumer has already installed. But what if the consumer targets ES2015+ browsers, or some platform (like Node) that doesn't require compilation at all (almost).

Bottom line is, I'd very much like to keep working on this, and please, feel free to come up with ideas and feature requests, or submitting pull requests (for instance with updates to dependencies, like you suggest), and I'll be available for discussion.

Thanks! 😃
